test/routes.js

```diff
@@ -30,6 +31,7 @@ describe('GET /', () => {
   it('it responds with 404', (done) => {
     chai.request(app).get('/')
       .end((err, res) => {
+        should.not.exist(err);
         res.should.have.status(404);
         done();
       });
```

Comment:
Nice! Thanks for adding that! Wow, did the linter catch that? Is it because `err` was not being used? (I'm new to node specs too!)

Reply:
Ah, I see in the output `Expected error to be handled`. 👍

Reply:
Yeah this was all caught by the packages you installled! Tbh this kind of stuff is exactly why I love working with linters, especially when there are autofixable rules like a lot of this PR is. 

These particular lines had to be manually added but it's always nice to get a good starting place to google from =D

Reply:
Ps there's definitely some stuff here like indentation that's stepping on Prettier's toes a bit, so if we add that they have an eslint plugin that quiets all the "code appearance" stuff from eslint.